<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddApertureWindow</class>
 <widget class="QDialog" name="AddApertureWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Aperture</string>
  </property>
  <layout class="QVBoxLayout" name="addApertureVerticalLayout">
   <item>
    <widget class="QLabel" name="apertureTitleLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add New Aperture</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="addApertureFormLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="widthLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="widthInput">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lengthLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="lengthInput">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="baseWidthLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Base Width</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="baseWidthInput">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="baseLengthLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Base Length</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="baseLengthInput">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="quantityLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Quantity</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="quantityInput">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="shapeLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Shape</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="DynamicComboBox" name="apertureShapeInput"/>
     </item>
     <item row="6" column="0">
      <widget class="ActivatorLabel" name="nibbleApertureLabel">
       <property name="styleSheet">
        <string notr="true">
                              QLabel {
                              padding: 3px;
                              }

                              QLabel[hovering=true] {
                              padding: 3px;
                              background-color: #d9d9d9;
                              border-radius: 5px;
                              }
                            </string>
       </property>
       <property name="text">
        <string>Nibble Aperture</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="DynamicComboBox" name="nibbleApertureInput"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="acceptButtons">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DynamicComboBox</class>
   <extends>QComboBox</extends>
   <header>widgets/DynamicComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ActivatorLabel</class>
   <extends>QLabel</extends>
   <header>widgets/ActivatorLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>acceptButtons</sender>
   <signal>accepted()</signal>
   <receiver>AddApertureWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>acceptButtons</sender>
   <signal>rejected()</signal>
   <receiver>AddApertureWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
